model/wire.cc

```diff
@@ -363,6 +365,15 @@ namespace
     }
     cairo_path_destroy (path);              
   }
+  
+  bool Wire::attachedToDefiningVar() const
+  {
+    auto t=to();
+    assert(t);             
+    if (auto i=dynamic_cast<IntOp*>(t->item().operationCast())) return i->intVar->varTabDisplay;
```

Comment:
Could we not override attachedToDefiningVar() in IntOp to implement this code there? That way, this code, and the code in Item::visible() is simplified.

Reply:
I'm busy fixing this now.